services/venice-server/src/main/java/com/linkedin/venice/server/VeniceServer.java

```diff
@@ -608,6 +609,7 @@ public void shutdown() throws VeniceException {
       if (exceptions.size() > 0) {
         throw new VeniceException(exceptions.get(0));
       }
+      LOGGER.info("Shutdown completed in {} ms", LatencyUtils.getLatencyInMS(startTimeMS));
```

Comment:
If any exception is thrown before this logging line, we may not able to see it in the log? If this log is important, we can consider using the `com.linkedin.venice.utils.Timer` which extends AutoCloseable for logging.